### PR TITLE
Update sentinels.tex

### PR DESCRIPTION
**Pull request by Silence711**
Hey guys :)
These are some slight changes (more like clarifications) I would add to the scenario. I'm not used to writing in code so you need to check the formatting. I love your work here and am very happy that my scenario has made it into the selection! Best regards
Silence

### CODE REVIEW

**Comment by qwrtln** [comment]
Hi! Thanks for helping keep your scenario up-to-date. This introduces a significant amount of text, so the layout is broken now. Let me see what I can do.

File: coops/sentinels.tex [hunk 1]
```diff
@@ -95,19 +95,24 @@ \subsection*{\MakeUppercase{Additional Rules}}
...
+      \item Capture a Mine or Settlement that is already Flagged.
+  \end{itemize}
 \end{itemize}
 
 \subsection*{\MakeUppercase{Enemy Spawning\\and Movement}}
 
 \begin{itemize}
   \item Enemy armies spawn on a random Obelisk field (use hero miniatures from factions not in play).
-  \item Determine which Obelisk field by rolling 2 Attack Dice:
+  \item Determine which Obelisk field by rolling 2 Attack Dice. Apply the results as following (D1/D2), reroll every 0:
```
Comment (tomaas-zeman) [inline coops/sentinels.tex:109]:
```suggestion
  \item Determine which Obelisk field by rolling 2 Attack Dice. Apply the results as follows (D1/D2), reroll any 0:
```

File: coops/sentinels.tex [hunk 2]
```diff
@@ -95,19 +95,24 @@ \subsection*{\MakeUppercase{Additional Rules}}
...
   \begin{itemize}[leftmargin=15pt]
-    \item First die: --1 west, +1 east, 0 reroll
-    \item Second die: --1 north, +1 south, 0 reroll
+    \item +1/+1 = North, +1/-1 = East
+    \item -1/+1 = West, -1/-1 = South
   \end{itemize}
-  \item For a more balanced distribution, in Rounds 7 and 9, the enemy spawns on the opposite side from the previous round.
-    Rounds 8 and 10 are entirely random.
+  \item For a more balanced distribution, in Rounds 7 and 9, only roll the second Attack Die (D2). For D1 take the opposite result from what was rolled in the previous Round.
```
Comment (tomaas-zeman) [inline coops/sentinels.tex:114]:
```suggestion
  \item For a more balanced distribution, in Rounds 7 and 9, only roll the second Attack Die (D2). For D1, take the opposite result from what was rolled in the previous Round.
```

**Comment by Silence711** [comment]
> Hi! Thanks for helping keep your scenario up-to-date. This introduces a significant amount of text, so the layout is broken now. Let me see what I can do.

Hi :) thank you very much! I'm not sure if the section about not being able to attack your ally or flag their mines is really necessary, but as it is included in the other coop scenarios I thought it might be good for being clear and consistent.

**Comment by qwrtln** [comment]
Here's the new layout:

en-05
en-06
en-07

**Review by qwrtln — APPROVED** [review]

**Review by tomaas-zeman — APPROVED** [review]

**Pull request merged**